python/tests/timeSeries/readUnrTxyz2_test.py

```diff
@@ -1,7 +1,21 @@
 import sys
 sys.path.append("./src")
 
-import timeSeries as ts
+from timeSeries import TimeSeries
```

Comment:
@twistmcflip outta curiosity, why this change? as opposed to the original `from timeSeries import timeSeries as ts`?

Reply:
Originally timeSeries was just a module. But @jaredonline changed it so that it is a module inside a package of the same name. I didn't realize that when I made the suggestion for the change.